schema.sql

```diff
@@ -11,4 +11,33 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals
-ADD species varchar(255);
\ No newline at end of file
+ADD species varchar(255);
+
+-- Multiple tables
+
+CREATE TABLE owners (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  full_name VARCHAR(255),
+  age  INT,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE species (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(255),
+  PRIMARY KEY (id)
+);
+
+ALTER TABLE animals
+DROP COLUMN species;
+
+ALTER TABLE animals 
+ADD COLUMN species_id INT;
+
+ALTER TABLE animals
+ADD CONSTRAINT fk_species
+FOREIGN KEY (species_id)
+REFERENCES species(id);
+
+ALTER TABLE animals 
+ADD COLUMN owner_id INT;
```

Comment:
- [ ] Good job so far but the column `owner_id` should have a foreign key referencing the `owners` table, please kindly fix that 🙃 🙃